### DISCUSSION TITLE
Support for Handshake domains

### DISCUSSION THREAD

**Discussion by chetankatke** [simple-login/app#713] [Feature Requests]
Visit https://handshake.org/ to know what is handshake domain
Please upvote so can we can have our TLD on simple login.

**Comment by ghost** [comment]
i'm totally against.
Since this is a mail services only public official TLD must be enabled.
Plus even if they would do that, i think this must be the last feature done, because of not enough user who actually use these Third party TLD

**Comment by gregbtm** [comment]
Was looking into this myself. Has this changed at all?

**Reply by chetankatke** [reply]
Sadly no
